README.md

```diff
@@ -0,0 +1,76 @@
+# Hello Microverse
+
+In this project, I set up a "Hello world" repository. No complex coding was required for this exercise. Our goal here is to master all of the tools and best practices we learned about in previous steps..
+
+
+
+## Builted With
+
+-GitHub
+
+-Git
+
+-Visual Code Studio
+
+-Node.js
+
+
+
+
+## Learning Objectives
+
+-Create branches with different base branches.
+
+-Understand the concept of version control.
+
+-Follow GitHub flow.
+
+-Open pull requests in GitHub.
+
+-Write descriptive, easy to understand commit messages.
+
+-Write short, detailed, easy to understand descriptions of pull requests.
+
+-Write a descriptive and easy to understand README in business English for every project submitted.
+
+-Demonstrate an ability to submit a project for code review.
+
+
+
+
+
+**For this project, we had to get familiar with Github Actions applying Linters to learn how to have cleaner and more profesional codes. Also we had to create a .gitignore file and practice adding new branches to the main repository.** 
+
+
+
+
+
+## I hope that it fulfill your espectations.
+
+
+
+
+
+👤 **Matias Aguirre**
+
+-GitHub: @lokurasrlz
+
+-LinkedIn: https://www.linkedin.com/in/matias-aguirre-ab51a623a/
+
+-Twitter: @lokurasrlz
+
+-Instagram: @aguirremati09

```

Comment:
- [OPTIONAL] I can see you are using the Microverse template, good job 👏I suggest adding your social media links to make a professional `README` file.

For Example You can use this template
```
- GitHub: [@username](Your_GitHub_Link)
- Twitter: [@username](Your_GitHub_Link)
- LinkedIn: [@username](Your_GitHub_Link)
```